js/samples/04.ai-apps/c.vision-cardGazer/package.json

```diff
@@ -9,31 +9,35 @@
         "dev:teamsfx": "env-cmd --silent -f .localSettings yarn dev",
         "dev": "nodemon --watch ./src --exec node --inspect=9239 --signal SIGINT -r ts-node/register ./src/index.ts",
         "build": "tsc --build",
-        "clean": "rimraf node_modules lib",
+        "clean": "npx rimraf node_modules lib",
         "lint": "eslint **/src/**/*.{j,t}s{,x} --fix --no-error-on-unmatched-pattern",
         "start": "tsc --build && node ./lib/index.js",
         "test": "echo \"Error: no test specified\" && exit 1",
         "watch": "nodemon --watch ./src -e ts --exec \"yarn start\""
     },
     "repository": {
         "type": "git",
-        "url": "https://github.com"
+        "url": "https://github.com/microsoft/teams-ai"
     },
     "dependencies": {
         "@microsoft/teams-ai": "~1.1.3",
         "@microsoft/teamsfx": "^2.3.1",
         "botbuilder": "^4.22.1",
         "dotenv": "^16.4.5",
+        "openai": "4.28.4",
         "replace": "~1.2.0",
         "restify": "~11.1.0"
     },
     "devDependencies": {
-        "@types/dotenv": "6.1.1",
-        "@types/restify": "8.5.12",
         "@types/node": "^20.12.5",
-        "ts-node": "^10.9.2",
+        "@types/restify": "8.5.12",
+        "@typescript-eslint/eslint-plugin": "^6.21.0",
+        "@typescript-eslint/parser": "^6.21.0",
         "env-cmd": "^10.1.0",
-        "nodemon": "~1.19.4",
```

Comment:
did you intend to remove this dev dep? I see it is still being used in the scripts as a command